Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -13,10 +13,9 @@
 #import <VungleAdsSDK/VungleAdsSDK.h>
 
 // Handle init callback for all adapter instances
-static ISConcurrentMutableSet<ISNetworkInitCallbackProtocol> *initCallbackDelegates = nil;
 static InitState initState = INIT_STATE_NONE;
```

Comment:
You should remove this. Even if it failed in the adapter, the SDK can maybe get it in the ads response

Reply:
You can also remove all the switch statements that uses this